src/app/regions/page.tsx

```diff
@@ -25,9 +25,9 @@ const Regions = () => {
         </h1>
       </header>
       <section className="grid md:grid-cols-2 lg:grid-cols-3 gap-4 px-2 lg:px-4 py-4 lg:py-8 max-w-7xl mx-auto">
-        {regionsData.map((region) => (
+        {regionsData.map((region, index) => (
           <Card
-            key={region.name}
+            key={index}
```

Comment:
**Question (non-blocking):** Why are we using the index instead of the name for the key?

Reply:
as of right now, we are using a bunch of the same region as placeholders - maybe we should revisit this in the future, but its also not priority since these are not dynamically moved and so it will have the same key